perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -379,7 +392,8 @@ sub form : Private {
 
     $form->process unless $form->processed;
 
-    $c->stash->{template} = $form->template || 'waste/index.html';
+    $c->stash->{template} = $form->template || 'waste/index.html'
+        unless $c->stash->{sent_confirmation_message}; # XXX?
```

Comment:
I think it is currently okay, in that we mandate an email address and so don't need to worry about phone verification; I just wasn't sure if this would work in all scenarios if that was allowed. But it might be fine.